api/app/src/main/kotlin/packit/repository/IndexRepository.kt

```diff
@@ -0,0 +1,8 @@
+package packit.repository
+
+import org.springframework.data.jpa.repository.JpaRepository
+import org.springframework.stereotype.Repository
+import packit.model.Packet
+
+@Repository
+interface IndexRepository: JpaRepository<Packet, Long>
```

Comment:
I guess we don't have detekt in here yet (would be a good idea to add that next!) - these files should all  have empty lines at the end.